scripts/train.py

```diff
@@ -297,7 +298,9 @@ def dummy_init_fn(module: torch.nn.Module) -> None:
                 cfg.load_path,
                 load_optimizer_state=not cfg.reset_optimizer_state,
                 load_trainer_state=not cfg.reset_trainer_state,
-                sharded_checkpointer=cfg.load_path_sharded_checkpointer,
+                # sharded_checkpointer=cfg.load_path_sharded_checkpointer,
+                sharded_checkpointer= False,
+                checkpoint_type=CheckpointType.unsharded
```

Comment:
Same question here

Reply:
I have added this because in `if cfg.load_path is not None:` we are passing the unsharded checkpoints which we download from r2. But `sharded_checkpointer=cfg.load_path_sharded_checkpointer,` this is taking sharded instead of unsharded. I had to manually set the flag to `False`.